### PR TITLE
[FIX] improvements rit upgrade

### PR DESCRIPTION
**Pull request by marcoscostazup**
**- What I did**
- Added Upgrade command to autocomplete
- Changed where upgrade warning appears (only at rit, rit --version and rit login)
- Remove cache usage when Upgrade command is called
- Added latest version on rit --version if tthere is a new version

**- How to verify it**
- Run 'rit upgrade'
- Run 'rit --version'
- Run 'rit'

**- Description for the changelog**
- Message of rit --version when there is a new version available
- Changed where upgrade warning appears (only at rit, rit --version and rit login)

### CODE REVIEW

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -63,9 +65,8 @@ var (
 	}
 
 	upgradeValidationWhiteList = []string{
```
Comment (viniciussousazup) [inline pkg/cmd/root.go:67]: "Whitelist" have different meaning on teamWhitelist and singleWhitelist

File: pkg/version/resolver.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 package version
 
 type Resolver interface {
-	StableVersion() (string, error)
+	StableVersion(fromCache bool) (string, error)
```
Comment (viniciussousazup) [inline pkg/version/resolver.go:4]: I thing StableVersion should always get value from cache, and if you need to get value without cache than:
//Delete Cache file
r.RemoveCache()
//Get stable version from cache if exist or create cache
r.StableVersion()

File: pkg/cmd/root.go (hunk 1 above)
Comment (marcoscostazup) [reply pkg/cmd/root.go:67]: Changed the name of the others

File: pkg/cmd/root.go [hunk 3]
```diff
@@ -204,6 +204,15 @@ func isWhitelist(whitelist []string, cmd *cobra.Command) bool {
...
 
 func versionFlag(edition api.Edition) string {
+	resolver := version.DefaultVersionResolver{
+		StableVersionUrl: StableVersionUrl,
+		FileUtilService:  fileutil.DefaultService{},
+		HttpClient:       &http.Client{Timeout: 1 * time.Second},
+	}
+	latestVersion, err := resolver.StableVersion(true)
+	if err == nil && latestVersion != Version {
+		return fmt.Sprintf(versionMsgWithLatestVersion, Version, edition, fmt.Sprintf(prompt.Yellow, fmt.Sprintf(latestVersionMsg, latestVersion)), BuildDate, runtime.Version())
```
Comment (viniciussousazup) [inline pkg/cmd/root.go:214]: too large line

Comment (marcoscostazup) [reply pkg/cmd/root.go:214]: Fixed!

File: pkg/version/version_test.go [hunk 4]
```diff
@@ -143,14 +147,24 @@ func TestDefaultVersionResolver_StableVersion(t *testing.T) {
 				FileUtilService:  tt.fields.FileUtilService,
 				HttpClient:       tt.fields.HttpClient,
 			}
-			got, err := r.StableVersion()
+			got, err := r.StableVersion(true)
```
Comment (viniciussousazup) [inline pkg/version/version_test.go:150]:
```suggestion
			got, err := r.StableVersion(tt.fields.fromCache)
```
and remove if !tt.fields.fromCache {

Comment (marcoscostazup) [reply pkg/version/version_test.go:150]: Great idea! Thanks

File: pkg/cmd/root.go [hunk 5]
```diff
@@ -51,7 +53,7 @@ var (
 		fmt.Sprintf("%s upgrade", cmdUse),
 	}
 
-	teamWhitelist = []string{
+	teamIgnorelist = []string{
```
Comment (erneliojuniorzup) [inline pkg/cmd/root.go:56]: The standard we use is whitelist, please go back.

Comment (marcoscostazup) [reply pkg/cmd/root.go:56]: Ok

**Pull request merged**